app/javascript/app/components/ndcs/ndcs-enhancements-map/ndcs-enhancements-map-selectors.js

```diff
@@ -296,7 +296,7 @@ export const getPreviousComparisonCountryValues = createSelector(
   [getPreviousComparisonIndicators, getISOCountries],
   (previousComparisonIndicators, isos) => {
     if (!previousComparisonIndicators) return null;
-    const previousComparisonCountryValues = {};
+    const previousComparisonCountryValues = [];
```

Comment:
This looks like an object. We are adding attributes to it

Reply:
Thank you for catching that.